### ISSUE TITLE
Initial validation loss twice the training loss

### ISSUE THREAD

**Issue by ghost** [nlpyang/PreSumm#223]
![image](https://user-images.githubusercontent.com/24579703/128281488-fbd03677-cec6-4faf-84cc-e935871f5d3c.png)

As you can see, the training loss start at 3.14 while 6.18 for the validation loss (i.e. validation loss is twice the training loss). It must have an error in the code. Not sure what it is yet. Apart from this detail, the code works well and the results are interesting.
